apps/rstudio/Dockerfile

```diff
@@ -0,0 +1,46 @@
+# syntax=docker/dockerfile:1
+FROM ubuntu:24.04
+SHELL ["/bin/bash", "-xe", "-o", "pipefail", "-c"]
+
+ARG APP_NAME
+ARG APP_VERSION
+
+WORKDIR /apps/${APP_NAME}
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get update -qy && \
+    apt-get install --no-install-recommends -qy \
+    software-properties-common dirmngr \
+    curl wget gpg gpg-agent libssl-dev libpq5 libclang-dev \
+    libnss3 libatk1.0-0 libatk-bridge2.0-0 libcups2 \
+    libgtk-3-0 libasound2t64 build-essential cmake file \
+    gfortran liblapack-dev libopenblas-dev libpng-dev \
+    libxkbcommon-x11-0 && \
+    wget -qO- https://cloud.r-project.org/bin/linux/ubuntu/marutter_pubkey.asc | tee -a /etc/apt/trusted.gpg.d/cran_ubuntu_key.asc && \
+    add-apt-repository "deb https://cloud.r-project.org/bin/linux/ubuntu $(lsb_release -cs)-cran40/" && \
+    apt-get install --no-install-recommends -y r-base r-cran-rstan r-cran-tidyverse && \
+    curl -sSO https://download1.rstudio.org/electron/jammy/amd64/rstudio-${APP_VERSION}-375-amd64.deb && \
```

Comment:
24.04 is not jammy.

Reply:
Good point but it's the link they put for all ubuntu versions of the official download page (and after checking this : https://download1.rstudio.org/ , no noble something) so who am i to argue ?

Reply:
it's the same indeed. All good I guess.

![Screenshot 2024-09-24 at 16-49-33 RStudio Desktop - Posit](https://github.com/user-attachments/assets/ef71973d-9cc7-4e2d-abc0-89d7ab96a417)